README.md

```diff
@@ -1,12 +1,15 @@
 # metric-hub
 
-Central hub for metric definitions that are considered the source of truth.
+metric-hub is the central source of truth for the definitions and metadata related to all of Mozilla's relevant business metrics. metric-hub is very much what is known as a ["semantic layer"](https://www.klipfolio.com/resources/data-stack/metrics-layer-vs-semantic-layer), in that the metric values themselves exist in the data warehouse, and the metric implementations exist in the data processing layer made of our [ETL infrastructure](https://github.com/mozilla/bigquery-etl/). metric-hub metric definitions can be thought of as pointers to the outputs of the canonical metric implementations, and it provides the abstraction layer that allows folks to access metric values without needing to know anything about the SQL required to extract those values from our warehouse.
+
+Some metrics may only be needed within a limited context, such as a value to be tracked over the course of an experiment (handled by [jetstream](https://github.com/mozilla/jetstream), our automated experiment analysis framework), or a piece of data to be ongoing monitored for one or more populations (via [OpMon](https://github.com/mozilla/opmon), an operational monitoring tool used to watch the impact of scaled roll-outs and to support certain product health dashboards). Other metrics are much more widely impactful, such as organizational KPIs and high level product success measurements.
+
 For more information on how to add metric definitions please see the [docs on docs.telemetry.mozilla.org](https://docs.telemetry.mozilla.org/concepts/metric_hub.html)
 
-All metric definitions can be referenced in other tooling and their configurations, such as, [jetstream](https://github.com/mozilla/jetstream)
-or [opmon](https://github.com/mozilla/opmon).
+All metric definitions can and should be referenced in other tooling and their configurations, including jetstream, OpMon, [BigQuery ETL](https://docs.telemetry.mozilla.org/concepts/metric_hub.html#using-metrics-in-etl-queries), and [python scripts](https://docs.telemetry.mozilla.org/concepts/metric_hub.html#using-metrics-in-python-scripts). All metrics are also made available in auto-generated Looker Explores, from which visualizations and dashboards can be generated.
 
 Jetstream specific configs or defaults can be added in the `jetstream/` directory.
 OpMon specific configs or defaults can be added in the `opmon/` directory.
+Looker Explore specific configs or defaults can be added in the `looker/` directory. This directory also supports the definitions of "statistics" (i.e. specific aggregations of metric-hub metrics), which will be materialized as measures in the autogenerated Explores. How to do this is explained in detail in the [documentation](https://docs.telemetry.mozilla.org/concepts/metric_hub.html#using-metrics-in-looker).
```

Comment:
Making this section a list could help with readability.  As it is all three sentences will get combined into a single paragraph.